packages/platform/atoms/calendar-settings/wrappers/CalendarSettingsPlatformWrapper.tsx

```diff
@@ -0,0 +1,240 @@
+                            </div>
+                          }>
+                          <div className="border-subtle border-t">
+                            <p className="text-subtle px-5 pt-4 text-sm">{t("toggle_calendars_conflict")}</p>
+                            <ul className="space-y-4 px-5 py-4">
+                              {connectedCalendar.calendars?.map((cal) => {
+                                return (
+                                  <PlatformCalendarSwitch
+                                    key={cal.externalId}
+                                    externalId={cal.externalId}
+                                    title={cal.name || "Nameless calendar"}
+                                    name={cal.name || "Nameless calendar"}
+                                    type={connectedCalendar.integration.type}
+                                    isChecked={cal.isSelected}
+                                    destination={cal.externalId === destinationCalendarId}
+                                    credentialId={cal.credentialId}
+                                  />
+                                );
+                              })}
+                            </ul>
+                          </div>
+                        </AppListCard>
+                      );
+                    }
+                    return (
+                      <Alert
+                        key={`alert-${connectedCalendar.credentialId}`}
+                        severity="warning"
+                        title={t("something_went_wrong")}
+                        message={<span>{t("calendar_error")}</span>}
+                        iconClassName="h-10 w-10 ml-2 mr-1 mt-0.5"
+                        actions={
+                          <div className="flex w-32 justify-end">
+                            <PlatformDisconnectIntegration
+                              credentialId={connectedCalendar.credentialId}
+                              trashIcon
+                              buttonProps={{ className: "border border-default" }}
+                              slug={connectedCalendar.integration.slug}
+                            />
+                          </div>
+                        }
+                      />
+                    );
+                  })}
+                </List>
+              </CalendarSettings>
+            );
+          }}
+        />
+      </div>
+    </AtomsWrapper>
+  );
+};
+
+const CalendarSettingsHeading = () => {
+  const { t } = useLocale();
+
+  return (
+    <div className="border-subtle border-b p-6">
+      <div className="flex items-center justify-between">
+        <div>
+          <h4 className="text-emphasis text-base font-semibold leading-5">{t("check_for_conflicts")}</h4>
+          <p className="text-default text-sm leading-tight">{t("select_calendars")}</p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const PlatformDisconnectIntegration = (props: {
```

Comment:
Would be good to split small components from this file into separate files.

Reply:
Since the smaller components are not being used anywhere else i.e. since they are exclusive to only the Platform wrappers thats why me and @ThyMinimalDev agreed its best to keep them in the same file itself.

Reply:
these components, at least currently don't really exist without each other, keeping in the same file for now should not be an issue